streaming-binlog/src/test/java/org/openmrs/analytics/FhirUriGeneratorTest.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * TODO: Add tests for the FhirEventWorker class; this requires refactoring FhirEventWorker and
+ * proper dependency injection for all external service calls that it does.
+ */
+public class FhirUriGeneratorTest
```

Comment:
Can we add some more unit-tests for `FhirUriGenerator`?

Reply:
Yes, I'll add a placeholder for this! I think FhirUriGenerator needs a total redesign to make it configurable

Reply:
Great; forgot to mark this as resolved before.